indico/modules/events/contributions/templates/display/contribution_display.html

```diff
@@ -0,0 +1,232 @@
+                                </li>
+                            {%- endfor %}
+                        </ul>
+                    </div>
+                </div>
+            {% endif %}
+        </div>
+
+        <div class="i-box-group vert">
+            <div class="i-box titled no-padding">
+                <div class="i-box-header">
+                    <div class="i-box-title icon-attachment">
+                        {% trans %}Materials{% endtrans %}
+                    </div>
+                    <div class="i-box-buttons toolbar right">
+                        <div class="group">
+                            {% if contribution.can_manage_attachments(session.user)
+                                or contribution.can_manage(session.user, role='submit') -%}
+                                <a href="#" class="i-button icon-edit"
+                                   data-locator="{{ contribution.locator|tojson|forceescape }}"
+                                   data-attachment-editor></a>
+                            {%- endif %}
+                        </div>
+                    </div>
+                </div>
+                <div class="i-box-content">
+                    <div class="attachments-box">
+                        {{ render_attachments(contribution.attached_items, management=false) }}
+                    </div>
+                </div>
+            </div>
+
+            <div class="i-box titled no-padding">
+                <div class="i-box-header">
+                    <div class="i-box-title">
+                        {% trans %}Subcontributions{% endtrans %}
+                    </div>
+                </div>
+                <div class="i-box-content">
+                    <ul class="group-list no-content-before">
+                        {% for subcontrib in contribution.subcontributions -%}
+                            <li>
+                                <span>
+                                    <a href="{{ url_for('.subcontribution_display', subcontrib) }}">{{ subcontrib.title }}</a>
+                                </span>
+                                {% if subcontrib.can_manage(session.user) -%}
+                                    <span class="right group">
+                                        <a href="#" class="icon-edit"
+                                           data-href="{{ url_for('.manage_edit_subcontrib', subcontrib) }}"
+                                           data-title="{% trans title=subcontrib.title %}Edit subcontribution '{{ title }}'{% endtrans %}"
+                                           data-ajax-dialog
+                                           data-reload-after></a>
+                                    </span>
+                                {%- endif %}
+                            </li>
+                        {%- else -%}
+                            <li>
+                                {% trans %}This contributions doesn't have any subcontributions{% endtrans %}
```

Comment:
Subcontribs are not that common; I wouldn't show the block at all if there aren't any.